src/nodedef.cpp

```diff
@@ -682,6 +682,8 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 	switch (drawtype) {
 	default:
 	case NDT_NORMAL:
+		material_type = (alpha == 255) ?
```

Comment:
`use texture alpha` means the alpha embedded in the texture is used, the 'alpha = ' nodedef is not.
But yes some mods may be using `alpha = ` in nodedef, however semi-translucency is not supported for normal nodes so i don't think we need to support it here, more opinions needed on this i think.